client/src/components/Query/Filter/index.jsx

```diff
@@ -26,49 +27,28 @@ export const createFilter = type => ({
 class Filter extends React.Component {
   constructor(props) {
     super(props);
+    const { dataSet = [], autoOpen, visible, sortData } = props;
```

Comment:
Please add selected

Reply:
As discussed, we'll wait for Lucas' input on this for more precisions on how the feature should behave.